module4/lessons/Week1/ReviewingConcepts.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: Oregon Trail Mini Project
+---
+
+Let's build a small application to practice what we've learned in Mods 1 and 2. Read throught the entire spec below before starting!
+
+## The Oregon Trail
+
+1. Fork and Clone [this starter repo](https://github.com/turingschool-examples/CaravanMVC)
+2. Open your forked repository in visual studio
+3. Build the solution - let you instructor know if you get any errors!
+
+
+### Building our Models
+
+1. Create a branch called `models`
+2. In your Caravan project, add classes outlined below.
+3. Commit after each class is complete - you should have 3 commits!
+4. When all three classes have been built, merge your `models` branch into your `main` branch.
+5. Push your main branch to GitHub.
```

Comment:
Can we have them create a PR to merge their branch into main that way? I would prefer to set the pattern that you never merge branches in visual studio, always with a PR.